mailmerge/template_message.py

```diff
@@ -71,12 +76,29 @@ def render(self, context):
         self._transform_recipients()
         self._transform_markdown()
         self._transform_attachments()
+        self._transform_attachment_references()
         self._message.__setitem__('Date', email.utils.formatdate())
         assert self._sender
         assert self._recipients
         assert self._message
         return self._sender, self._recipients, self._message
 
+    def _make_attachment_content_id_header(self, filepath):
```

Comment:
This might make more sense a stand-alone function.  Then, when it's called later, add the ID to the data structure.  It's not intuitive that this function would modify `self._attachment_content_ids`.